utils/convert.go

```diff
@@ -0,0 +1,8 @@
+package utils
+
+import "math"
+
+func ConvertBytes(value *int64) int64 {
+	mbSize := int64(float64(*value) / math.Pow(float64(1024), float64(2)))
```

Comment:
All of these casts are unnecessary. Consider the following:
```
return (*value)/(1<<20)
```

Also, the name of this function is very ambiguous. You could rename it.

The utils package seems to be of little value. Consider moving stuff to where it is used.